### DISCUSSION TITLE
Can we have the release date in the listing as well?

### DISCUSSION THREAD

**Discussion by robertgzr** [community/community#6608] [Other Feature Feedback, Questions, & Ideas] [answered]
What lead me to come here was that I'm unable to see the release date without clicking 'Read more'

The other bit (although less important to me) would be to have the `<n> commits to <branch> since this release`. Both of those are a metric I use to figure out how recent a release is and it would be nice to not have to click another time to get to them :)

**Comment by MylesBorins** [comment, marked as answer]
Hey @robertgzr not showing the date was a bug and is now fixed.

Re: the `n` commits to `branch` since this release feature would definitely be worth revisiting... we removed it as it was inconsistent in the past (it only showed comparison to `master` which wasn't always accurate).
